example/src/views/HomeView.vue

```diff
@@ -864,6 +945,29 @@ const textFields = ref([
       </rui-text-field>
     </div>
   </div>
+
+  <div :class="css.section">
+    <h2 class="text-h4 mb-6" data-cy="alerts">Alerts</h2>
+    <div class="grid gap-4 grid-rows-2 grid-cols-3">
+      <rui-alert
+        v-for="(alert, i) in alerts"
+        :key="i"
+        v-bind="alert"
+        :action-text="alert.clicks !== undefined && 'Action'"
+        :on-action="
+          alert.clicks !== undefined &&
+          (() => {
+            alert.clicks++;
+          })
+        "
+        :title="`${alert.type} ${
+          alert.clicks !== undefined ? `(${alert.clicks})` : ''
+        } ${alert.closed ? '(Closed)' : ''}`"
+        :on-close="alert.closeable && (() => (alert.closed = true))"
```

Comment:
can we replace `on-action` and `on-close` with events emitted from the alert component and then use `@action=` and `@close=` event handlers

Reply:
other than this, it's all good.